libcudacxx/include/cuda/std/__functional/bind.h

```diff
@@ -299,10 +298,10 @@ class __bind_r : public __bind<_Fp, _BoundArgs...>
 public:
   typedef _Rp result_type;
 
-  template <class _Gp,
-            class... _BA,
-            class = __enable_if_t<is_constructible<_Fd, _Gp>::value
-                                  && !is_same<__libcpp_remove_reference_t<_Gp>, __bind_r>::value>>
+  template <
+    class _Gp,
+    class... _BA,
+    class = __enable_if_t<is_constructible<_Fd, _Gp>::value && !is_same<remove_reference_t<_Gp>, __bind_r>::value>>
```

Comment:
```suggestion
    class = enable_if_t<is_constructible<_Fd, _Gp>::value && !is_same<remove_reference_t<_Gp>, __bind_r>::value>>
```

Reply:
Have you seen https://github.com/NVIDIA/cccl/pull/2763/commits/899126a5046121f41c50c94fa3e7656f6dead01b

`__enable_if_t` only ever instantiates 2 specializations whereas `enable_if_t` instantiates one per use